assignments/prototypes.js

```diff
@@ -18,10 +18,14 @@
 function GameObject(attributes) {
   this.createdAt = attributes.createdAt;
   this.dimensions = attributes.dimensions;
-  this.destroy = function () {
-    return `Object was removed from the game.`;
-  };
+
 }
+
+function destroy () {
```

Comment:
Remember to bind the function to the constructor we use prototype:

>GameObject.prototype.destroy = function () {
  return `Object was removed from the game.`;
};